portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,16 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div id="title">
+            <h1>Welcome to the Portfolio of Andrés Abundis</h1>
+        </div>
```

Comment:
Can you please align the divs and check the indentation? 

Indentation should be 2 spaces.(https://google.github.io/styleguide/htmlcssguide.html#Indentation)

Reply:
Done!